locale/fr.po

```diff
@@ -9966,13 +9963,11 @@ msgstr "Échec de la création de %s « %s »."
 #~ msgid "Print preview"
 #~ msgstr "Aperçu avant impression"
 
-#, fuzzy
 #~ msgid "&Preview..."
-#~ msgstr " Aperçu"
+#~ msgstr " Aperçu..."
```

Comment:
Just for the record, I don't think it's useful to modify the commented out entries, they're not used anyhow and, in fact, should probably be all removed -- AFAIK gettext tools keep them there just to allow reusing the old translations in the new ones, but I don't think it's very useful in practice (unless tools like Poedit use them too?).